### ISSUE TITLE
Separate pixel scaling factor for sprite preview

### ISSUE THREAD

**Issue by GordemProo** [aseprite/aseprite#2862]
I like how Aseprite looks with double sized pixels, but it is annoying when you couldn't look at you sprite in 1:1 resolution – it blurs with screen scaling at 200%. When screen and UI are 100%, UI is too small. When screen is 100% and UI is 200%, UI and editing area look _very_ inconsistent. Could there be a separate option for sprite preview's screen scaling, so editing area, regular UI are consistent with each other, and you can see the sprite 1:1 not blurred? Like that:
![new-screen-scaling-preferences-mockup](https://user-images.githubusercontent.com/80538841/127781801-21a35cea-eda6-493b-a1b0-ec6dd1c4a365.png)
Compared:
![aseprite-separate-scaling-difference](https://user-images.githubusercontent.com/80538841/127901160-969e58c6-86d5-440d-b43b-1d8677c823a9.png)
Consider cursor' and selection's pixels' size.

**Comment by JamesO2x** [comment]
If you click inside the Preview window, you can press the number keys at the top of the keyboard to Zoom the preview window independently. Pressing the [1] key will zoom to 1x, [2] will give you 2x zoom, etc.

This should solve your issue. I don't think adding a pixel scale option in the preferences would be necessary, since we can zoom the Preview window with hotkeys on the fly.

I hope this helps! :)

**Comment by GordemProo** [comment]
@JamesO2x
No, it's all about the resolution with which the sprite is rendered. If it's the same for the editing area and the preview, (as it is in present), either the interface is too small and it's hard to see and point the cursor, or it's impossible to see the sprite in 1:1 resolution __(1:1 regarding _screen_ pixel size, _not program_)__ without blurring.
Pixel Scaling is _program_ pixel size divided by _screen_ pixel size. That's why blurring happens – _program_ pixels are too large to display four, two times smaller sprite pixels.

**Comment by Gasparoken** [comment]
Hi @GordemProo, thank you for reporting!
In other words, I think you said:
"Why can't the Preview reproduce a non-blurred image with 200% Screen Scaling and 100% UI Scaling Elements in certain Preview zoom condition? A higher resolution is available, so there should be a way to decouple the Preview Scaling from the Screen Scaling."
(By the way, the same thing happens to the Editor view port)

Screen Scaling is a simple treatment to enlarge all the pixels on the screen. That's why we get blurred images with an apparent available high resolution.
The real solution to this should be 100% Screen Scaling and 200% UI Scaling Elements, but you said:
"UI and editing area look very inconsistent"
We'd appreciate it if you could tell us what you mean by this to keep improving Aseprite!

**Comment by Gasparoken** [comment]
Hi @GordemProo. You said:
"When screen is 100% and UI is 200%, UI and editing area look very inconsistent."
We'd appreciate it if you could tell us what you mean by this to keep improving Aseprite!

**Comment by GordemProo** [comment]
@Gasparoken

#### “In other words, I think you said: […]”
Yes, that is *exactly* that!

#### “Inconsistent”
The borders and especially the font are wide and bold, while the canvas border, the cursor and (especially) the black and white selection border are much thinner. That's what confuses and messes me.
I think it's just me. There is nothing impartial, really.
I don't insist.

**Comment by Gasparoken** [comment]
When the editor has a 100% screen scale and 200% UI scale:
The cursor -I agree- is too thin on my screen (13") and hard to find.
The edge of the canvas is thinner than others, I don't see any problem here.
The selection marching ants, maybe could be thicker to improve visibility.
The font is the same as 200% screen and 100% UI.
You saw font inconsistency and thick widgets VS ultra thin cursor, canvas border and selection marching ants.
I think we can improve this.
Thanks @GordemProo for your advise!

**Comment by Gasparoken** [comment]
This will solved via #2869

**Comment by dacap** [comment]
Hi there 👋  this might be related to #1506 and we might need to specify a special scale for the preview (each window can has its own scale, at least in v1.3). I'll leave the issue open just to remember that the option could be added in some ways (screen scale for preview + effects)
